.github/workflows/do-not-merge.yml

```diff
@@ -0,0 +1,16 @@
+name: Do not merge
+
+on:
+  pull_request:
+    branches:
+      - 8.x-[1-9]+.x
+
+jobs:
+  do-not-merge:
+
+    runs-on: ubuntu-latest
+
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'DO NOT MERGE') }}
+
+    steps:
+      - run: false
```

Comment:
I am not sure about this. I think its more relevant to quickly see, if your branch fails or not. The existance of the tag should be enough to make us not merge it.

Reply:
Will remove it